dashboard/serializers.py

```diff
@@ -25,6 +25,25 @@ def serialize_enrollments(cls, enrollments):
         return [{'title': course.title} for course in enrolled_courses]
 
     @classmethod
+    def count_courses_passed(cls, program, mmtrack):
+        """
+        Returns number of passed courses.
+
+        Args:
+            program (Program): A program object.
+            mmtrack (dashboard.utils.MMTrack): a instance of all user information about a program
+        Returns:
+            num_complete_courses: A number of passed courses.
+        """
+        num_courses_passed = 0
+        for course in program.course_set.all():
+            course_run = course.first_unexpired_run()
+            if course_run and mmtrack.has_passed_course(course_run.edx_course_key):
+                num_courses_passed += 1
+
+        return num_courses_passed
```

Comment:
I agree that `first_unexpired_run()` was the wrong function but I'd rather not do any heavy refactoring in this PR. We should use the output of `get_info_for_course` so that the decision on whether a course is passed or not is made in one place. A refactor would be good in a future PR though, I just prefer to keep the refactoring and feature work separate to make reviewing easier and to reduce the possibility of bugs slipping through